packages/core/useResize/index.ts

```diff
@@ -0,0 +1,336 @@
+  }
+
+  function onPointerMove(evt: PointerEvent) {
+    isPathIncludesTarget.value = (evt.composedPath() as Element[]).includes(target.value!)
+
+    pointer.currentX = evt.x
+    pointer.currentY = evt.y
+
+    if (evt.pressure === 0 && !evt.movementX && !evt.movementY)
+      handlePointer(pointer)
+
+    if (!isOverEdge.value || !isResizing.value)
+      return
+
+    let newWidth = width
+    let newHeight = height
+
+    const xDiff = Math.abs(evt.x - pointer.startX) * unref(xMultiplier)
+    const yDiff = Math.abs(evt.y - pointer.startY) * unref(yMultiplier)
+
+    if (direction.value.includes('bottom'))
+      newHeight += (evt.y > pointer.startY ? yDiff : -yDiff)
+
+    if (direction.value.includes('top'))
+      newHeight += (evt.y > pointer.startY ? -yDiff : yDiff)
+
+    if (direction.value.includes('left'))
+      newWidth += (evt.x > pointer.startX ? -xDiff : xDiff)
+
+    if (direction.value.includes('right'))
+      newWidth += (evt.x > pointer.startX ? xDiff : -xDiff)
+
+    onResizeMove.trigger({
+      pointer: evt,
+      xDiff,
+      yDiff,
+      startX: pointer.startX,
+      startY: pointer.startY,
+      newWidth: clamp(newWidth, unref(minWidth as number), unref(maxWidth as number)),
+      newHeight: clamp(newHeight, unref(minHeight as number), unref(maxHeight as number)),
+      setSize,
+    })
+
+    if (!unref(disableResize))
+      setSize(newWidth, newHeight)
+
+    evt.preventDefault()
+  }
+
+  async function onPointerDown(evt: PointerEvent) {
+    if (evt.pointerType === 'touch') {
+      pointer.currentX = evt.x
+      pointer.currentY = evt.y
+      isPathIncludesTarget.value = (evt.composedPath() as Element[]).includes(target.value!)
+      await nextTick()
+    }
+    if (isOverEdge.value || (isOverEdge.value && evt.pointerType === 'touch')) {
+      ({ width, height } = target.value!.getBoundingClientRect())
+      isResizing.value = true
+      pointer.startY = evt.y
+      pointer.startX = evt.x
+      onResizeStart.trigger({ pointer: evt })
+      evt.preventDefault()
+    }
+    else {
+      isOutside.value = true
+    }
+  }
+
+  const isOnForeground = (x: number, y: number, corner = false, xReduce?: number, yReduce?: number) => {
+    const getCorner = (xA: number, yA: number) => {
+      let found = false
+      while (!found && Math.abs(xA - x) < unref(borderRadius) / 2) {
+        xA += xReduce!
+        yA += yReduce!
+        found = target.value!.contains(window!.document.elementFromPoint(xA, yA)) || window!.document.elementFromPoint(xA, yA) === target.value!
+      }
+      return found
+    }
+    const element = window!.document.elementFromPoint(x, y)
+    return isPathIncludesTarget.value || (element && target.value!.contains(element)) || (corner && getCorner(x, y))
+  }
+
+  const isEdgeActive = (edge: Edges) => {
+    return unref(edges).includes(edge)
+  }
+
+  function handlePointer({ currentX, currentY }: typeof pointer) {
+    if (isResizing.value || isOutside.value)
```

Comment:
Should also check if `target.value` exists here since `useEventListener` will be on the top level